include/unifex/let_value.hpp

```diff
@@ -227,16 +228,16 @@ struct _op<Predecessor, SuccessorFactory, Receiver>::type {
   template <typename Operation, typename... Values>
   friend struct _successor_receiver;
 
-  template <typename SuccessorFactory2, typename Receiver2>
+  template <typename Predecessor2, typename SuccessorFactory2, typename Receiver2>
```

Comment:
Same as above in `let_done` and `let_error`.  I think lines 378-379 are hand-rolling the `member_t<Self, Predecessor>` type alias with this:
```
decltype((static_cast<Sender&&>(sender).pred_)
```

Reply:
> lines 378-379 are hand-rolling the member_t<Self, Predecessor> type alias with this:
 
 Yep, I noticed this too, and added a test for "l value connectable"-ness though nothing had to be changed for this test to pass.